pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sphinx_gherkindoc"
-version = "3.2.2"
+version = "3.2.4"
```

Comment:
I just went ahead and anticipated #3 being merged first. However the merge order goes, something will have to be changed.

Reply:
I hope #10 gets merged before 3. reminds me I need to dink the version in 10.